### PR TITLE
Bimamba

### PR DESCRIPTION
**Pull request by smm477**
updated modernbert architecture, imputation output in bed ranges, fixed loss functions and masking

### CODE REVIEW

File: src/python/bimamba/bimamba_model.py [hunk 1]
```diff
@@ -165,34 +158,33 @@ def __init__(self, input_dim, d_model, num_classes, n_layer, d_conv=4, device=No
         self.bimamba = BiMamba(d_model, n_layer, d_conv=d_conv, **factory_kwargs)
         self.classification_head = nn.Sequential(
             nn.Linear(d_model, d_model * 2),
-            # nn.Softplus(),
+            #nn.Softplus(),
```
Comment (zrm22) [inline src/python/bimamba/bimamba_model.py:161]: Should this comment be removed?
